app/models/puget_sound_custom_alert_feed.rb

```diff
@@ -0,0 +1,18 @@
+class PugetSoundCustomAlertFeed < AlertFeed
+  def fetch
+    # no-op
+    super
+  end
+
+  def add_alert_item(title, summary, url)
```

Comment:
Remember this method the next time we implement an AlertFeed class that needs to manually create alerts; we may benefit from moving this to the base class.